src/assets/content.tsx

```diff
@@ -146,79 +140,37 @@ const content: Content = {
   whatsNew: {
     title: 'Was ist neu?',
     description: [
-      `Wir haben die Farben angepasst, damit der Wasserbedarf der Berliner Bäume intuitiver zu erfassen ist. Es beginnt bei einem trockenem Gelb und geht bis in ein sattes Grün.<br /><div style="padding-top:0.5rem;padding-bottom:0.5rem; display:flex">
-      <div
-        style=" background-color:#fde725; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#fde725"
-      ></div>
-      <div
-        style=" background-color:#cae11f; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#cae11f"
-      ></div>
-      <div
-        style=" background-color:#95d840; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#95d840"
-      ></div>
-      <div
-        style=" background-color:#63cb5f; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#63cb5f"
-      ></div>
-      <div
-        style=" background-color:#3bbb75; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#3bbb75"
-      ></div>
-      <div
-        style=" background-color:#22a884; width: 13px; height: 13px; border-radius: 100px; margin-right: 5px;"
-        color="#22a884"
-      ></div>
-    </div> Ebenfalls haben wir unsere Pumpen-Daten geupdated. Die kommen jetzt jede Woche frisch von Open Street Maps. `,
-      `Ihr könnt sehen ob Pumpen: "funktionsfähig" <span
-       style=
-       "display: inline-block;
-        background-color: ${workingColor.hex};
-        width: 13px;
-        height: 13px;
-        border-radius: 100px;
-        margin-right: 5px;"
+      `Der Frühling kann kommen! Wir haben unser Backlog aufgeräumt, einige Funktionen verbessert und neue Funktionen hinzugefügt. Die wichtigsten Verbesserungen im Überblick:<br />
+      <div style="padding-top:0.5rem;padding-bottom:0.5rem; display:flex">
+        <div>
+          <img style="max-width:18px; margin-right: 5px;" alt="Geolocate Icon" src="/images/geolocate-icon.png"/><b>Geolokalisierung</b><br>
+          Mit einem Klick auf das Icon am unteren linken Bildschirmr  and zum eigenen Standort navigieren.
+        </div>
+      </div>
+      <div style="padding-top:0.5rem;padding-bottom:0.5rem; display:flex">
+        <div>   
+          <span style="display:inline-block; background-color:#75ADE8; width: 16px; height: 16px; border-radius: 100px; margin-right: 4px;"></span>
```

Comment:
Feels like you inserted a dot because of the lack of an icon for that topic

Reply:
found it nice and helpful for reading as it should give the user a more visual appeal.
colors of the the two dots are corresponding to what the feauteres are doing/about
blue -->  same like new water colour of map
green --> same like treetop of our gdk logo

Reply:
meh. Pretty far fetched ;)

Reply:
:D maybe @dnsos has some opinion here?
I am not sticky at all

Reply:
Hm, since we use dots as markers for trees, it makes sense to use it for the tree routing. The blue dot for map style is a bit off then. But to be honest, I don't think it would lead to confusion

Reply:
alright, thank you. then let's resolve this discussion :)